code/game/objects/items/grenades/chem_grenade.dm

```diff
@@ -5,6 +5,7 @@
 /obj/item/grenade/chem_grenade
 	name = "chemical grenade"
 	desc = "A custom made grenade."
+	var/casedesc = "This basic model accepts both beakers and bottles. It heats contents by 10°K upon ignition." // Appears when examining empty casings.

```

Comment:
I don't know if it's in CONTRIBUTING but I think we've been pushing to place all `var/` after parent overrides.

Reply:
Would it be fine if I move it to line 22 after `var/no_splash = FALSE` then?

Reply:
Yes